services/user/Homepage/ui/src/components/nav-user.tsx

```diff
@@ -0,0 +1,341 @@
+                            />
+                            <span>{connectedAccount}</span>
+                          </DropdownMenuItem>
+                        ))}
+                        <DropdownMenuSeparator />
+                        <DropdownMenuItem
+                          onClick={() => {
+                            login();
+                          }}
+                        >
+                          <PlusCircle className="mr-2 h-4 w-4" />
+                          <span>More...</span>
+                        </DropdownMenuItem>
+                      </DropdownMenuSubContent>
+                    </DropdownMenuPortal>
+                  </DropdownMenuSub>
+                </DropdownMenuGroup>
+              )}
+              <DropdownMenuGroup>
+                <DropdownMenuSub>
+                  <DropdownMenuSubTrigger>
+                    <Moon className="mr-2 h-4 w-4" />
+                    <span>Theme</span>
+                  </DropdownMenuSubTrigger>
+                  <DropdownMenuPortal>
+                    <DropdownMenuSubContent>
+                      <DropdownMenuItem onClick={() => setTheme("light")}>
+                        <Sun className="mr-2 h-4 w-4" />
+                        <span>Light</span>
+                      </DropdownMenuItem>
+                      <DropdownMenuItem onClick={() => setTheme("dark")}>
+                        <Moon className="mr-2 h-4 w-4" />
+                        <span>Dark</span>
+                      </DropdownMenuItem>
+                      <DropdownMenuSeparator />
+                      <DropdownMenuItem onClick={() => setTheme("system")}>
+                        <Sun className="mr-2 h-4 w-4" />
+                        <span>System</span>
+                      </DropdownMenuItem>
+                    </DropdownMenuSubContent>
+                  </DropdownMenuPortal>
+                </DropdownMenuSub>
+              </DropdownMenuGroup>
+              <DropdownMenuSeparator />
+              <DialogTrigger asChild>
+                <DropdownMenuItem
+                  disabled={!currentUser}
+                  onClick={() => {
+                    generateInvite.mutate();
+                  }}
+                >
+                  <User className="mr-2 h-4 w-4" />
+                  Create invite
+                </DropdownMenuItem>
+              </DialogTrigger>
+
+              <DropdownMenuItem
+                disabled={!canExportAccount}
+                onClick={() => {
+                  window.location.href = siblingUrl(
+                    undefined,
+                    "auth-sig",
+                    undefined,
+                    false,
+                  )
+                }}>
+                <Download className="mr-2 h-4 w-4" />
+                Export account
+              </DropdownMenuItem>
+
+              <DropdownMenuSeparator />
+              <DropdownMenuItem onClick={() => onLogout()}>
+                <LogOut />
```

Comment:
`<LogOut className="mr-2 h-4 w-4" />`

Reply:
Also, this menu item should be hidden (along with the separator above it) when the user's already logged out.